docs/docsite/rst/collections_guide/collections_installing.rst

```diff
@@ -7,6 +7,11 @@ Installing collections
 
   If you install a collection manually as described in this paragraph, the collection will not be upgraded automatically when you upgrade the ``ansible`` package or ``ansible-core``.
 
+Installing collections in containers
+------------------------------------
+
+You can install collections with their dependencies in containers known as execution environments. See :ref:`getting_started_ee_index` for details.
```

Comment:
@Andersson007 Is it maybe a bit too soon to add a link to the EE getting started just yet? If we decide to move that to a different spot then we'll need to update here as well, right?

Reply:
@oraNod i would start now not waiting for possible moves. If it happens, can we just `grep/sed` and change it everywhere at once?

Reply:
yeah, we can def do that. it's not a big deal.

Reply:
I replaced the ref with the URL, PTAL